modules/seqexec/server/src/main/scala/seqexec/server/gnirs/GnirsControllerEpics.scala

```diff
@@ -187,7 +187,7 @@ object GnirsControllerEpics extends GnirsController {
   private def setOtherCCParams(config: Other): List[SeqAction[Unit]] = {
     val open = "Open"
     val focus = "best focus"
-    val wavelengthTolerance = 0.0005
+    val wavelengthTolerance = 0.0001
```

Comment:
Where is this used?

Reply:
If you're using a double and rounding then you might want to use `gem.math.Wavelength` which is exact.

Reply:
@cquiroz It is used in only one place, in the call to `smartSetDoubleParam` at the end of the same function.

Reply:
@tpolecat `wavelengthTolerance` is not a wavelength, it is a dimensionless quantity, the relative threshold to decide if the wavelength parameter needs to be applied.

Reply:
There is a dimension there though, if I understand it correctly.  You compare one wavelength to another to see if it differs sufficiently to change?  If they were `Wavelength`s then we could state exactly how many angstroms different that they need to be.

Reply:
@swalker2m It is relative, like in `(current - requested)/requested > threshold`. Therefore, it has no units.